cherrypicks.go

```diff
@@ -135,3 +141,136 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
+		}
+	}
+	return false
+
+}
+
+func cherryPickToBranch(log *logrus.Entry, comment *github.IssueCommentEvent, pr *github.PullRequest, conf *config, targetBranch string) error {
+
+	if !branchIsABranchInRepository(log, comment.GetRepo().GetName(), targetBranch) {
+		return fmt.Errorf("%s is not a branch in %s", targetBranch, comment.GetRepo().GetName())
+	}
+
+	prBranchName := fmt.Sprintf("cherry-%s-%s",
+		targetBranch, pr.GetHead().GetLabel())
+	state, err := git.Commands(
+		git.Command("init", "."),
+		git.Command("remote", "add", "mendersoftware",
+			"git@github.com:mendersoftware/"+comment.GetRepo().GetName()+".git"),
+		git.Command("fetch", "mendersoftware"),
+		git.Command("checkout", "mendersoftware/"+targetBranch),
+		git.Command("checkout", "-b", prBranchName),
+	)
+	defer state.Cleanup()
+	if err != nil {
+		return err
+	}
+
+	if _, err = git.Command("cherry-pick", "-x",
+		pr.GetHead().GetRef()+" ^"+pr.GetBase().GetRef()).
+		With(state).Run(); err != nil {
+		if strings.Contains(err.Error(), "conflict") {
+			return errorCherryPickConflict
+		}
+		return err
+	}
+	if _, err = git.Command("push", "mendersoftware", prBranchName+":"+prBranchName).
+		With(state).Run(); err != nil {
+		return err
+	}
+
+	newPR := &github.NewPullRequest{
+		Title: github.String(fmt.Sprintf("[Cherry %s]: %s",
+			targetBranch, comment.GetIssue().GetTitle())),
+		Head: github.String(prBranchName),
+		Base: github.String(targetBranch),
+		Body: github.String(
+			fmt.Sprintf("Cherry pick of PR: %d\nFor you %s :)",
+				pr.GetID(), comment.Sender.GetName())),
+		MaintainerCanModify: github.Bool(true),
+	}
+	client := clientgithub.NewGitHubClient(conf.githubToken)
+	_, err = client.CreatePullRequest(
+		context.Background(),
+		githubOrganization,
+		comment.GetRepo().GetName(),
+		newPR)
+	if err != nil {
+		return fmt.Errorf("Failed to create the PR for: (%s) %v",
+			comment.GetRepo().GetName(), err)
+	}
+	return nil
+}
+
+func cherryPickPR(log *logrus.Entry, comment *github.IssueCommentEvent, pr *github.PullRequest, conf *config, body string, githubClient clientgithub.Client) error {
+	targetBranches := parseCherryTargetBranches(body)
+	if len(targetBranches) == 0 {
+		return fmt.Errorf("No target branches found in the comment body: %s", body)
+	}
+	conflicts := make(map[string]bool)
+	for _, targetBranch := range targetBranches {
+		if err := cherryPickToBranch(log, comment, pr, conf, targetBranch); err != nil {
+			if err == errorCherryPickConflict {
+				conflicts[targetBranch] = true
+				continue
+			}
+			log.Errorf("Failed to cherry pick: %s to %s, err: %s",
+				comment.GetIssue().GetTitle(), targetBranch, err)
+		}
+	}
+	// Comment with cherry links on the PR
+	commentText := `Hi :smileycat:
+I did my very best, and this is the result of the cherry pick operation:
+`
+	for _, targetBranch := range targetBranches {
+		if !conflicts[targetBranch] {
+			commentText = commentText +
+				fmt.Sprintf("\t* %s:white_check_mark:\n", targetBranch)
```

Comment:
For consistency, just would add an space
```suggestion
				fmt.Sprintf("\t* %s :white_check_mark:\n", targetBranch)
```